### DISCUSSION TITLE
Recommended camera that works best out of the box?

### DISCUSSION THREAD

**Discussion by pedro-fleek** [jakobwilm/slstudio#60] [Q&A]
Hi,

I've been playing around with the project for a couple weeks and I have to say that I am super impressed with it so far!

However, I seem to be running into issues with the XIMEA camera that I have been trying.  Would anyone have any suggestions as to which models of the supported cameras has the best chance of working out of the box?

Any advice appreciated.  

Thanks,

Pedro

**Comment by jakobwilm** [comment]
Hello Pedro. As mentioned in the Readme, the ximea cameras of that series are tested and will work well out of the box.
